client/me/security-ssh-key/security-ssh-key.tsx

```diff
@@ -57,6 +60,9 @@ export const SecuritySSHKey = () => {
 		onSuccess: () => {
 			dispatch( recordTracksEvent( 'calypso_security_ssh_key_add_success' ) );
 			dispatch( successNotice( __( 'SSH key added to account.' ), noticeOptions ) );
+			if ( queryArgs.source && queryArgs.siteSlug ) {
+				page.redirect( `/${ queryArgs.source }/${ queryArgs.siteSlug }` );
```

Comment:
I'm not sure an automatic redirect would be expected behavior. Did you see something like this elsewhere?

Alternatively, what do you think about showing some form of message on the settings page when `queryArgs.source` and `queryArgs.siteSlug` are set, and the message could have a link back to the Hosting Configuration page?

Reply:
No, I just did that as the issue said "redirect the user back to hosting configuration" 
I can change that.
Regarding the other comment, it's a matter of preference. It could be done like that as well.
I'll check it out.

Reply:
What do you thing about that?
![back-to-ssh](https://user-images.githubusercontent.com/497103/200279688-677d4742-7219-4b91-937a-f614e43a081f.png)

Reply:
> I just did that as the issue said "redirect the user back to hosting configuration"

@kozer Sorry, a misunderstanding I suppose. I said "there's an obvious way for them to get back to 'Hosting Configuration' after they've added the SSH key." and could've been more specific that I was referring to UI.

> What do you think about that?

This is pretty great! I think we could go one step further and only have one back link in the top left, labeled as: "Back to Hosting Configuration"

Reply:
I'm with @danielbachhuber on this and it's used throughout the app so we rename the navigation based on where we are coming from. "Back to Hosting Configuration" make sense.